portality/static/js/doaj.js

```diff
@@ -1,165 +1,165 @@
 /** base namespace for all DOAJ-specific functions */
 var doaj = {
-    init : function() {
```

Comment:
Is this all commented out as part of your workflow, and that it's all coming back in again when you are done?